packages/eyevinn-skin/src/components/buttons/volume/VolumeSlider.tsx

```diff
@@ -19,17 +19,8 @@ export default function VolumeSlider({ muted, onClick, toggleMute, value }) {
       onMouseMove={onMouseMove}
       onMouseLeave={onMouseLeave}
       onClick={() => {
-        let newVolumePercentage = mousePos;
-        if (mousePos < 5) {
-          newVolumePercentage = 0;
-          if (!muted) {
-            toggleMute();
-          }
-        }
-        else if (mousePos > 95) {
-          newVolumePercentage = 100;
-        }
-        if (muted) {
+        let newVolumePercentage = mousePos < 5 ? 0 : mousePos > 95 ? 100: mousePos;
+        if((muted && newVolumePercentage) || (!muted && !newVolumePercentage)) {
```

Comment:
Maybe make it more obvious by comparing to 0 rather than check whether it is falsy.

i.e. `(muted && newVolume > 0) || (!muted && newVolume === 0)` is more obvious why we should toggle imho 🤔